<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerWindow</class>
 <widget class="QMainWindow" name="CustomerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
     QWidget {
         background-color: #f0f0f0;
     }
     QLineEdit, QPushButton, QTextEdit {
         border: 1px solid #ccc;
         border-radius: 5px;
         padding: 5px;
     }
     QPushButton {
         background-color: #4CAF50;
         color: white;
     }
     QLabel#statusLabel, QLabel#registrationErrorLabel {
         font-weight: bold;
     }
     QLabel#registrationRequirementsLabel {
         font-size: 12px;
         color: #555;
     }
     QLabel#registrationErrorLabel {
         color: red;
     }
    </string>
   </property>
   <widget class="QLineEdit" name="projectNameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Project Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Register Project</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="projectIdLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
    <property name="placeholderText">
     <string>Project Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
